packages/bierzo-wallet/src/routes/welcome/index.tsx

```diff
@@ -1,30 +1,71 @@
-import { Theme } from '@material-ui/core';
+import { Avatar, Button, Theme } from '@material-ui/core';
 import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import Image from 'medulas-react-components/lib/components/Image';
+import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
+import avatar from '../../assets/iov-logo.svg';
 import { history } from '../../store/reducers';
 import { PAYMENT_ROUTE } from '../paths';
 
 const useStyles = makeStyles((theme: Theme) => ({
   welcome: {
-    backgroundColor: theme.palette.primary.main,
+    backgroundColor: theme.palette.background.default,
     width: '100vw',
-    height: '100vh',
+    height: 'auto',
+    minHeight: '100vh',
     display: 'flex',
+    flexDirection: 'column',
     placeItems: 'center',
     placeContent: 'center',
   },
+
+  avatarRoot: {
+    backgroundColor: theme.palette.primary.main,
+    width: '200px',
+    height: '200px',
+  },
+
+  avatarIcon: {
+    width: '50%',
+  },
+
+  heading: {
+    margin: '40px 0 40px 0',
+    fontSize: '20px',
+  },
+
+  buttonRoot: {
```

Comment:
Be consistent. Remove the `Root` suffix 😉.

Also, specify the width of a button, does not seem to be natural, I suggest to wrap the button with a Block and use the width prop `<Block width="150px">...</Block>`, in this way you can remove this CSS rule.

Reply:
Button has inherent "width: auto". By wrapping it with `<Block width="150px">`, this happens:
![imagen](https://user-images.githubusercontent.com/44572727/56949604-a5c27700-6b33-11e9-91bb-67ae18e0c9aa.png)

Reply:
The same happens if I remove the width and height of the avatar class and wrap the avatar with a block of 200x200, it has a default size of 40x40 so this happens:
![imagen](https://user-images.githubusercontent.com/44572727/56950262-5bda9080-6b35-11e9-9797-ba2580cad8ba.png)